src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte

```diff
@@ -48,8 +58,75 @@
 	const close = () => dispatch('icClose');
 	const back = () => dispatch('icSendBack');
 
-	// TODO: implement send function when related services are ready
-	const send = () => {};
+	const send = async () => {
+		const network = nonNullish(networkId) ? mapNetworkIdToBitcoinNetwork(networkId) : undefined;
+
+		if (isNullish(network)) {
+			toastsError({
+				msg: { text: $i18n.send.error.no_btc_network_id }
+			});
+			return;
+		}
+
+		if (isNullishOrEmpty(destination)) {
+			toastsError({
+				msg: { text: $i18n.send.assertion.destination_address_invalid }
+			});
+			return;
+		}
+
+		if (invalidAmount(amount) || isNullish(amount)) {
```

Comment:
from the code `invalidAmount` already checks for nullish

Reply:
Somehow TS requires to have additional isNullish check. I guess for the same reason it is done like this in IcSendTokenWizard.